runtimes/peregrine/src/kilt/mod.rs

```diff
@@ -100,3 +106,32 @@ impl pallet_asset_switch::Config<KiltToEKiltSwitchPallet> for Runtime {
 	#[cfg(feature = "runtime-benchmarks")]
 	type BenchmarkHelper = crate::benchmarks::asset_switch::CreateFungibleForAssetSwitchPool1;
 }
+
+parameter_types! {
+	pub const NativeAsset: Location = Junctions::Here.into_location();
+}
+
+impl pallet_bonded_coins::Config for Runtime {
+	type BaseDeposit = runtime_common::constants::bonded_coins::BaseDeposit;
```

Comment:
I think the assets for the bonding curve should go in the same module. So if the bonded coins config is implemented here, I would move over the bonded assets config here as well.

Reply:
The new assets deployment solely exists for bonding curve functionality, so to me it feels fair that they go together, in whatever module.